hazelcast/src/main/java/com/hazelcast/config/CardinalityEstimatorConfig.java

```diff
@@ -43,6 +46,18 @@
      */
     public static final int DEFAULT_ASYNC_BACKUP_COUNT = 0;
 
+    /**
+     * The default merge policy used for cardinality estimators
+     */
+    public static final MergePolicyConfig DEFAULT_MERGE_POLICY_CONFIG =
+            new MergePolicyConfig(HyperLogLogMergePolicy.class.getName(), MergePolicyConfig.DEFAULT_BATCH_SIZE);
+
+    private static final String[] ALLOWED_POLICIES = {
```

Comment:
Oh, I just see that this prevents to write a custom merge policy for HyperLogLog. I know that this is a very special data structure, but is this wanted?

Reply:
Its a very special data structure indeed :)
Its certainly an extra step than what we did for others, but given the circumstances I would even argue, that we shouldn't be having this configurable at all. So I don't see how this hurts.